jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/MavenProjectProperties.java

```diff
@@ -187,8 +201,10 @@ public JibContainerBuilder createJibContainerBuilder(
       throws IOException {
     try {
       if (isWarProject()) {
-        Path explodedWarPath =
-            Paths.get(project.getBuild().getDirectory(), project.getBuild().getFinalName());
+        Build build = project.getBuild();
+        Path war = Paths.get(build.getDirectory(), build.getFinalName() + ".war");
```

Comment:
Do we need to check if this file exists?

Reply:
Somehow I couldn't make a comment here. Copying it over from below:

For checking the file, I think we're good. The same happens for the intermediate exploded WAR directory. We catch IOException and suggest to do mvn package.